.evergreen/config.yml

```diff
@@ -1391,7 +1395,7 @@ buildvariants:
       - name: ".test !.enterprise-auth !.snappy !.zstd"
 
   - matrix_name: "tests-42-lts-zlib-zstd-support"
-    matrix_spec: { version: ["4.2", "latest"], os-ssl-32: "*" }
+    matrix_spec: { version: ["4.2", "4.4", "latest"], os-ssl-32: "*" }
     display_name: "${version} ${os-ssl-32}"
     tasks:
       - name: ".test !.enterprise-auth !.snappy"
```

Comment:
Just curious: why don't we test snappy on 4.2+?

Reply:
Not really sure. Seems like we only test snappy on 3.4, which was the first version to support it. I'm inclined to leave it as-is because I don't see too much value in testing snappy against other versions.

Reply:
I would like to understand your reasoning, but let's not debate it on this PR, as it was more of a drive-by comment.